config/plugins/visualizations/vtk/config/vtk.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="VTK" embeddable="true">
+    <description>Visualization Toolkit</description>
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test test_attr="ext" result_type="datatype">plyascii</test>
+            <test test_attr="ext" result_type="datatype">plybinary</test>
+            <test test_attr="ext" result_type="datatype">vtpascii</test>
+            <test test_attr="ext" result_type="datatype">vtpbinary</test>
+            <to_param param_attr="id">dataset_id</to_param>
+        </data_source>
+    </data_sources>
+    <params>
+        <param type="dataset" var_name_in_template="hda" required="true">dataset_id</param>
+    </params>
+    <requirements>
+        <requirement type="npm" version="0.0.1" package="@galaxyproject/vtk"/>
+    </requirements>
+    <entry_point entry_point_type="script" src="dist/index.js" />
+    <tests>
+        <test>
+            <param name="dataset_id" value="http://cdn.jsdelivr.net/gh/galaxyproject/galaxy-test-data/square.plyascii" />
+        </test>
+        <test>
+            <param name="dataset_id" value="http://cdn.jsdelivr.net/gh/galaxyproject/galaxy-test-data/horse.plybinary" />
+        </test>
+        <test>
+            <param name="dataset_id" value="http://cdn.jsdelivr.net/gh/galaxyproject/galaxy-test-data/square.vtpascii" />
+        </test>
+        <test>
+            <param name="dataset_id" value="http://cdn.jsdelivr.net/gh/galaxyproject/galaxy-test-data/human.vtpbinary" />
+        </test>
+    </tests>
+    <help>
```

Comment:
Should we stick to this syntax?

https://github.com/galaxyproject/galaxy/pull/18722/files#diff-7de0ac81b4f0d1aaacb3e7a7843cf56a90d9e6b6a728d2a5318a84fe5a1b5b03R11

I guess we don't want to implement rst? But maybe then enfocring here the explicity markdown annotation is a good thing?

Reply:
I don't think that we need a `format` attribute right now since we are only supporting `markdown` and it will remain the default. We can introduce it down the road if we start supporting other formats.

Reply:
@bgruening ok, I added the format attribute and cdata for consistency. Thank you!